query-engine/connector-test-kit-rs/query-tests-setup/src/datamodel_rendering/mod.rs

```diff
@@ -30,13 +30,20 @@ pub trait DatamodelRenderer {
 }
 
 /// Render the complete datamodel with all bells and whistles.
-pub fn render_test_datamodel(config: &TestConfig, test_database: &str, template: String) -> String {
+pub fn render_test_datamodel(
+    config: &TestConfig,
+    test_database: &str,
+    template: String,
+    excluded_features: &[&str],
+) -> String {
     let tag = config.test_connector_tag().unwrap();
+    let excluded_features: Vec<_> = excluded_features.iter().map(|f| format!(r#""{}""#, f)).collect();
     let all_features = GENERATOR
         .active_features()
         .iter()
         .chain(GENERATOR.hidden_features())
         .map(|f| format!(r#""{}""#, f))
+        .filter(|f| !excluded_features.contains(f))
```

Comment:
This is hacky and ugly. I hope you don't mind, I needed a quick & dirty way to disable preview features.

Reply:
I think this is fine. Maybe just put it into a separate function to compute the feature flags that should be enabled.